src/spyglass/linearization/merge.py

```diff
@@ -1,8 +1,8 @@
 import datajoint as dj
 
-from spyglass.linearization.v0.main import (
+from spyglass.linearization.v0.main import (  # noqa F401
```

Comment:
This is the kind of thing that flops back and forth if we're not all on the same `black`. Maybe you're the first to edit this file since the update to `25.X`